### DISCUSSION TITLE
20220902 일갼회고

### DISCUSSION THREAD

**Discussion by dhjefflee** [codestates/BEB-05-Team-Vote#40] [일간회고]
오늘은 어떻게 프로젝트에 기여했나요?
타입스크립트를 공부했고 supabase를 공부했습니다. 
팀원들과 함께 하부르타 DAO 서비스 활성화 방안을 논의 했습니다. 

오늘의 프로젝트에서 힘든 점은 무엇인가요?
타입스크립트 적용이 아직 쉽지 않고 공부해야할 일들이 많습니다. 
아이가 아파 병원 갈일이 많은데 시간조정이 어려워 팀내 프로젝트 참여에 힘든점이 많았습니다. 

내일은 프로젝트에 기여하기 위해 무엇을 해야 하나요?
 다른 타입스크립트 사용한 프로젝트 보고 공부하겠습니다.
 supabase를 공부하고 초기 세팅을 시도해야할것 같습니다. 
데이타 베이스 설계를 해야할것 같습니다.

**Comment by thyoondev** [comment]
조셉! 화이팅입니다!!   
그리고 템플릿 등록해놓으시면 회고 작성하는데 더 편하실거에요~     
[회고 템플릿 등록 방법](https://endurable-dodo-552.notion.site/3cd11a1383ef419193b242b8343ee4ce)

**Reply by dhjefflee** [reply]
항상 친절하고 자세한 가이드 감사합니다~ ^^
